tests/integration-tests/tests/schedulers/test_slurm_accounting/test_slurm_accounting/pcluster.config.update.yaml

```diff
@@ -28,14 +28,12 @@ Scheduling:
             - InstanceType: c5.xlarge
             - InstanceType: m5.xlarge
             - InstanceType: c5d.xlarge
-            - InstanceType: c6i.xlarge
+            - InstanceType: t3.xlarge
           MinCount: 0
           MaxCount: 10
       Networking:
         SubnetIds:
-          {% for private_subnet_id in private_subnet_ids %}
```

Comment:
I agree on removing the multi-subnet from a test that is not focused on that to keep it simple. 
However, why is multi subnet a problem now?